extension/tasks/dependabotV2/utils/dependabot-cli/DependabotOutputProcessor.ts

```diff
@@ -95,16 +96,21 @@ export class DependabotOutputProcessor implements IDependabotUpdateOutputProcess
           return true;
         }
 
+        // TODO: Add config for custom PR description header text? Task V1 supported this via DEPENDABOT_MESSAGE_HEADER
+        // TODO: Add config for custom PR description footer text? Task V1 supported this via DEPENDABOT_MESSAGE_FOOTER
+        // TODO: Add config for GitHub compatibility score badges? Task V1 supported this via DEPENDABOT_COMPATIBILITY_SCORE_BADGE
+
```

Comment:
I do not think we need to make these customizable at all.

Reply:
```suggestion
```